skimage/draw/draw.py

```diff
@@ -125,7 +125,7 @@ def circle(r, c, radius, shape=None):
     return ellipse(r, c, radius, radius, shape)
 
 
-def set_color(img, coords, color):
+def set_color(img, coords, color, alpha=None):
```

Comment:
I disagree, this makes the readability of the documentation much harder. If you decide you want to change it later on, just change it from alpha=1 to alpha=None for 2 versions and check for `alpha is None` to raise a deprecation warning.